cmake/Platform/Linux/PackagingPostBuild_linux.cmake

```diff
@@ -10,8 +10,15 @@ file(REAL_PATH "${CPACK_SOURCE_DIR}/.." LY_ROOT_FOLDER)
 include(${LY_ROOT_FOLDER}/cmake/Platform/Common/PackagingPostBuild_common.cmake)
 include(${CPACK_CODESIGN_SCRIPT})
 
-file(${CPACK_PACKAGE_CHECKSUM} ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}.deb file_checksum)
-file(WRITE ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}.deb.sha256 "${file_checksum}  ${CPACK_PACKAGE_FILE_NAME}.deb")
+if($ENV{O3DE_PACKAGE_TYPE} STREQUAL "SNAP")
+    add_custom_command(OUTPUT ${CPACK_TOPLEVEL_DIRECTORY}/O3DE_${O3DE_INSTALL_VERSION_STRING}_amd64.snap.assert
+        COMMAND snapcraft sign-build ${CPACK_TOPLEVEL_DIRECTORY}/O3DE_${O3DE_INSTALL_VERSION_STRING}_amd64.snap > ${CPACK_TOPLEVEL_DIRECTORY}/O3DE_${O3DE_INSTALL_VERSION_STRING}_amd64.snap.assert
+        VERBATIM
+    )
+elseif($ENV{O3DE_PACKAGE_TYPE} STREQUAL "DEB")
```

Comment:
Before you merge, will need to make sure that this is added to the build pipeline script here: https://github.com/o3de/o3de/blob/development/scripts/build/Platform/Linux/build_config.json#L360